### DISCUSSION TITLE
proposal: ship flake8heavened module

### DISCUSSION THREAD

**Discussion by pwoolvett** [flakeheaven/flakeheaven#35] [Ideas]
Both of these would be solved if flakeheaven ships a `flake8heavened` module

#32
#17

**Comment by jose-rojas0** [comment]
Can you think of any downsides to adding such a module? It seems like it would improve usability, so then this sounds like a good idea to me

**Comment by pwoolvett** [comment]
:thinking: That I can think of, if there was a name clash with another package! (eg somebody else shipped a `flake8heavened` module/pkg) :rofl: 

I think (99%) poetry should support this, via `include_extra` or similar...
